enterprise/api/v1/serializers.py

```diff
@@ -329,6 +329,32 @@ class Meta:
         )
 
 
+class EnterpriseCourseEnrollmentWithAdditionalFieldsReadOnlySerializer(EnterpriseCourseEnrollmentReadOnlySerializer):
+    """
+    Serializer for EnterpriseCourseEnrollment model with additional fields.
+    """
+
+    class Meta:
+        model = models.EnterpriseCourseEnrollment
+        fields = (
+            'enterprise_customer_user',
+            'course_id',
```

Comment:
`enterprise_customer_user` and `course_id` are duplicated on the next 2 lines.

Reply:
Good catch! Removed.